fastlane/lanes/release.rb

```diff
@@ -504,19 +504,29 @@
 # @param [Boolean] beta Indicate if we should build a beta or regular release
 #
 def trigger_buildkite_release_build(branch:, beta:)
-  build_url = buildkite_trigger_build(
-    buildkite_organization: BUILDKITE_ORGANIZATION,
-    buildkite_pipeline: BUILDKITE_PIPELINE,
-    branch: branch,
-    environment: { BETA_RELEASE: beta },
-    pipeline_file: 'release-builds.yml',
-    message: beta ? 'Beta Builds' : 'Release Builds'
-  )
+  environment = {
+    IS_BETA_RELEASE: beta,
+    RELEASE_VERSION: release_version_current
+  }
 
-  return unless is_ci
+  # If we're running on CI, we can directly start the release pipeline jobs within the same build
+  if is_ci
+    buildkite_pipeline_upload(
+      pipeline_file: BUILDKITE_RELEASE_PIPELINE,
+      environment: environment
+    )
+  else
+    build_url = buildkite_trigger_build(
+      buildkite_organization: BUILDKITE_ORGANIZATION,
+      buildkite_pipeline: BUILDKITE_PIPELINE,
+      branch: branch,
+      environment: environment,
+      pipeline_file: ,
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/2da8fb6267631eb8c8416b70907c34b9846f10a4/fastlane/lanes/release.rb#L524"></span>Style/HashSyntax: Include the hash value.</td>
    </tr>
  </tbody>
</table>